hide/comp/SceneEditor.hx

```diff
@@ -2038,4 +2038,13 @@ class SceneEditor {
 		}
 		return null;
 	}
+
+	public function projectMeshSprays() {
+		var sprays : Array<hrt.prefab.l3d.MeshSpray> = [];
+		for(e in sceneData)
+			if(Std.is(e, hrt.prefab.l3d.MeshSpray))
+				sprays.push(cast e);
```

Comment:
We need a more recursive lookup - sprays are not necessarily at toplevel. OTOH we don't want to lookup into MeshSpray so a custom recursive algorithm is necessary.

Reply:
fixed